FPSCamera/FPSCamera.csproj

```diff
@@ -17,6 +17,8 @@
       {HintPathFromItem};
       C:\Program Files (x86)\Steam\steamapps\common\Cities_Skylines\Cities_Data\Managed\;
       ..\..\..\..\..\SteamLibrary\steamapps\common\Cities_Skylines\Cities_Data\Managed\;
+      C:\Program Files (x86)\Steam\steamapps\common\Cities_Skylines\Cities_Data\Managed\;
```

Comment:
I don't find the purpose of this duplicate lines. Could you elaborate?

Reply:
I made a mistake